addOns/oast/src/main/java/org/zaproxy/addon/oast/services/interactsh/InteractshService.java

```diff
@@ -176,14 +176,26 @@ public void register() throws InteractshException {
             HttpMessage reqMsg = new HttpMessage(reqHeader, reqBody);
             httpSender.sendAndReceive(reqMsg);
             if (reqMsg.getResponseHeader().getStatusCode() != 200) {
+                LOGGER.error(
```

Comment:
The error level should be reserved just to potential bugs in ZAP, warn would be preferable.